pkg/config/config.go

```diff
@@ -242,11 +242,15 @@ func fillOpenShiftRouteSuffixIfNecessary(nonCachedClient client.Client, configMa
 func ConfigMapPredicates(mgr manager.Manager) predicate.Predicate {
 	return predicate.Funcs{
 		UpdateFunc: func(evt event.UpdateEvent) bool {
-			updateConfigMap(mgr.GetClient(), evt.ObjectNew)
+			if evt.ObjectNew.GetName() == "devworkspace-controller-configmap" {
+				updateConfigMap(mgr.GetClient(), evt.ObjectNew)
```

Comment:
well, let's rename updateConfigMap because it's confusing. Like `loadConfigFromCM`

Reply:
Good point -- we're not really watching anything there anymore.